src/App.tsx

```diff
@@ -10,33 +11,34 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTab] = useState('tab-1');
+
+  useEffect(() => {
+    if (!tabs.map(tab => tab.id).includes(selectedTabId)) {
+      setSelectedTab('tab-1');
+    }
+  }, [selectedTabId]);
+
+  const onTabSelected = (newTab: {
+    id: string;
+    title: string;
+    content: string }) => {
+    setSelectedTab(newTab.id);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
```

Comment:
this can be writer easier as a template literal string, e.g.

<img width="311" alt="image" src="https://github.com/mate-academy/react_tabs/assets/30219274/970bb0c8-f1fc-46c2-ab7a-091f5c8d08bf">